itou/job_applications/tests/tests.py

```diff
@@ -862,7 +862,7 @@ def test_email_deliver_approval(self):
         assert approval.user.get_full_name() in email.subject
         assert approval.number_with_spaces in email.body
         assert approval.start_at.strftime("%d/%m/%Y") in email.body
-        assert approval.end_at.strftime("%d/%m/%Y") in email.body
+        assert str(approval.remainder.days) in email.body
```

Comment:
```suggestion
        assert f"{approval.remainder.days} jours" in email.body
```
pour éviter que ce nombre ne soit en fait dans le code postal ou le numéro de téléphone